src/compiler/types.ts

```diff
@@ -8756,6 +8756,7 @@ namespace ts {
         readonly includeAutomaticOptionalChainCompletions?: boolean;
         readonly includeCompletionsWithInsertText?: boolean;
         readonly includeCompletionsWithClassMemberSnippets?: boolean;
+        readonly includeCompletionsWithObjectLiteralMethodSnippets?: boolean;

```

Comment:
Maybe it’s good to have this anyway, but I was wondering if `includeCompletionsWithSnippetText` is enough to keep from sending this to VS—since snippet support seems to be the primary reason VS can’t use these right now.

Reply:
This still works without `includeCompletionsWithSnippetText`, it just means the body of the method we suggest won't have a tab stop. Should that not be the behavior? I don't think I've given much thought to the experience in the absence of snippet support when doing the class member snippet feature, because I was mainly thinking about vscode. But now that I think about it, it's pretty annoying not having the tab stop...

Reply:
I think personally I would have expected it to be predicated on snippet support. But if we decided the opposite on class member signature completions, we should stay consistent.